packages/template/__tests__/emit-content.test.ts

```diff
@@ -1,4 +1,5 @@
-import { SafeString } from '@glimmer/runtime';
+// @ts-expect-error cannot import, but we only need the type...
+import type { SafeString } from '@glimmer/runtime';
```

Comment:
Why did it stop working? It seems to be here: https://github.com/glimmerjs/glimmer-vm/blob/v0.89.0/packages/%40glimmer/runtime/index.ts#L43

And I don't think we would want `@ts-expect-error` here, if the only reason we want to import it is for the type, but specifically the type import wasn't working, what does this even do? (making it `any`, probably)

Reply:
This was the error:
https://github.com/typed-ember/glint/actions/runs/7712187814/job/21019163076#step:9:139